elastic/beats
# Add integration tests for cloudwatch

Test the following:

- Deploy a lambda that watch cloudwatch log group
- Remove an existing lambda
- processors defined in a function works.
- global process works
- fields defined in a function works.
- fields globally defined works.
- Make sure concurrency, timeout, memory limit and description are
correctly set on the lambda.

Tests requires the following:

- CLOUD_ID in the environment
- CLOUD_AUTH in the environment
- AWS credentials available in the environment
- Environment variable `FUNCTIONBEAT_INTEGRATION_TESTS` is set.

Fixes: #8850
